cluster/sync.sh

```diff
@@ -42,8 +42,8 @@ pods_ready_wait() {
   wait_failed=''
   max_retries=12
   retry_counter=0
-  echo "Waiting for kubemacpool leader container to be ready ..."
-  while [[ "$(./cluster/kubectl.sh wait pod -n kubemacpool-system --for=condition=Ready -l kubemacpool-leader=true --timeout=1m)" = $wait_failed ]] && [[ $retry_counter -lt $max_retries ]]; do
+  echo "Waiting for kubemacpool recourses to be ready ..."
+  while [[ "$(./cluster/kubectl.sh wait pod -n kubemacpool-system --all --for=condition=Ready --timeout=1m)" = $wait_failed ]] && [[ $retry_counter -lt $max_retries ]]; do
```

Comment:
`--all` means all resources, you are waiting for `pod` specifically, so you can drop `--all`

Reply:
actually it's mandatory, since I want to hit both pods without specific name, but I think doing it by label will be better. something like
`kubectl.sh wait pod -l app=kubemacpool -n kubemacpool-system --for=condition=Ready --timeout=1m`